backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/StubRenderer.kt

```diff
@@ -6,18 +6,32 @@
 package org.jetbrains.kotlin.backend.konan.objcexport
 
 import org.jetbrains.kotlin.descriptors.ClassDescriptor
+import org.jetbrains.kotlin.descriptors.DeclarationDescriptor
+import org.jetbrains.kotlin.descriptors.DeclarationDescriptorWithSource
+import org.jetbrains.kotlin.psi.KtDeclaration
+import org.jetbrains.kotlin.psi.KtPrimaryConstructor
+import org.jetbrains.kotlin.resolve.source.getPsi
+import org.jetbrains.kotlin.backend.common.serialization.extractSerializedKdocString
 
 object StubRenderer {
     fun render(stub: Stub<*>): List<String> = collect {
         stub.run {
+            val kDocString = this.descriptor?.extractKDocString()
             this.comment?.let { comment ->
                 +"" // Probably makes the output more readable.
                 +"/**"
+                kDocString?.let {
+                    + it.removeSurrounding("/**", "*/")
+                    +"" // Probably makes the following comment more readable.
+                }
                 comment.contentLines.forEach {
                     +" $it"
                 }
                 +"*/"
             }
+                    // Let's try to keep non-trivial kdoc formatting intact, when there is no other comments here
```

Comment:
Actually, your explanation in https://github.com/JetBrains/kotlin-native/pull/4441#discussion_r558416537 is quite convincing, and if the related tooling (e.g. documentation tooltips in Xcode) can properly handle two comments (could you please check this?), then we probably should prefer keeping the kdoc formatting always intact by emitting two comments.

Reply:
As I can see, XCode 12 (at least) does handle two HeaderDoc comments correctly. Particularly in my sample the exported kdoc rendered as "Summary" in Quick Help (without any tag it goes as `@brief` by default) and `@note` section rendered as Note, as expected. 
So I'd prefer the most conservative approach here, i.e. avoiding any reformatting of KDoc whenever possible.

Reply:
Totally agree.